filestack/mixins/imagetransformation.py

```diff
@@ -117,13 +117,13 @@ def quality(self, value=None):
     def zip(self):
         return self._add_transform_task('zip', locals())
 
-    def fallback(self, handle=None, cache=None):
+    def fallback(self, file=None, cache=None):
```

Comment:
Why are we changing from handle to file?

Reply:
We have **file** parameter instead of **handle** in the filestack sdk of fallback function.